pkg/apis/upgrade.cattle.io/constants.go

```diff
@@ -15,6 +15,9 @@ const (
 	// LabelController is the name of the upgrade controller.
 	LabelController = GroupName + `/controller`
 
+	// LabelExclusive is the value of the Exclusive flag.
```

Comment:
No tautological doc comments, please - don't just say that it is what it is, but explain what it's for.
```suggestion
	// LabelExclusive is set if the plan should not run concurrent with other plans.
```